scipy/signal/_filter_design.py

```diff
@@ -5102,22 +5102,23 @@ def _design_notch_peak_filter(w0, Q, ftype, fs=2.0):
     return b, a
 
 
-def iircomb(w0, Q, ftype='notch', fs=2.0):
+def iircomb(w0, Q, ftype='notch', fs=2.0, pass_zero=False):
```

Comment:
```suggestion
def iircomb(w0, Q, ftype='notch', fs=2.0, *, pass_zero=False):
```

Reply:
+1 I think we should do this as we add more parameters